docs/api.md

```diff
@@ -142,6 +142,45 @@ devices.
 
 **NOTE:** These are subject to change.
 
+#### raid
+
+This field contains the information about the RAID configuration for bare 
+metal servers. 
+
+The sub-fields are
+
+* *hardwareRAIDVolumes* -- It contains the list of logical disks for hardware 
+  RAID, if rootDeviceHints isn't used, first volume is root volume. Furthermore,
+  it defines the desired configuration of volume in hardware RAID.
+  * *level* -- 'RAID level for the logical disk. The following levels are 
+    supported: 0;1;2;5;6;1+0;5+0;6+0.'
+  * *name* -- Name of the volume. Should be unique within the Node. If not 
```

Comment:
The term Node has a different meaning in k8s, let's use "server" or "machine".

Reply:
okay sure.